corehq/apps/app_manager/app_schemas/case_properties.py

```diff
@@ -18,117 +18,106 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_forms(app):
+    """
+    Return list of forms in the app
+    """
+    if app.doc_type == 'RemoteApp':
+        return []
+    forms = []
+    for module in app.get_modules():
+        for form in module.get_forms():
+            forms.append(form)
+    return forms
+
+
 class ParentCasePropertyBuilder(object):
 
     def __init__(self, app, defaults=(), per_type_defaults=None):
         self.app = app
         self.defaults = defaults
         self.per_type_defaults = per_type_defaults or {}
 
-    @property
+    def _get_relevant_apps(self, include_shared_properties):
+        apps = [self.app]
+        if self.app.case_sharing and include_shared_properties:
+            for app in self._get_other_case_sharing_apps_in_domain():
+                apps.append(app)
```

Comment:
nit: these two lines could be reduced to `apps.extend(self._get_other_case_sharing_apps_in_domain())`